<ui version="4.0" stdsetdef="1" >
  <author></author>
  <comment></comment>
  <exportmacro></exportmacro>
  <class>CSSCustomDialog</class>
  <widget class="QWidget" name="CSSCustomDialog" >
    <property name="geometry" >
      <rect>
        <x>0</x>
        <y>0</y>
        <width>584</width>
        <height>456</height>
      </rect>
    </property>
    <property name="whatsThis" stdset="0" >
      <string>&lt;b>Stylesheets&lt;/b>&lt;p>See http://www.w3.org/Style/CSS for further information on cascading style sheets.&lt;/p></string>
    </property>
    <layout class="QGridLayout" >
      <item row="0" column="1" >
        <widget class="Q3GroupBox" name="GroupBox3" >
          <property name="sizePolicy" >
            <sizepolicy>
              <hsizetype>5</hsizetype>
              <vsizetype>3</vsizetype>
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
            </sizepolicy>
          </property>
          <property name="title" >
            <string>Font Size</string>
          </property>
          <layout class="QGridLayout" >
            <item row="0" column="1" >
              <widget class="QComboBox" name="basefontsize" >
                <property name="sizePolicy" >
                  <sizepolicy>
                    <hsizetype>3</hsizetype>
                    <vsizetype>1</vsizetype>
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                  </sizepolicy>
                </property>
                <property name="editable" >
                  <bool>true</bool>
                </property>
                <item>
                  <property name="text" >
                    <string>7</string>
                  </property>
                </item>
                <item>
                  <property name="text" >
                    <string>8</string>
                  </property>
                </item>
                <item>
                  <property name="text" >
                    <string>9</string>
                  </property>
                </item>
                <item>
                  <property name="text" >
                    <string>10</string>
                  </property>
                </item>
                <item>
                  <property name="text" >
                    <string>11</string>
                  </property>
                </item>
                <item>
                  <property name="text" >
                    <string>12</string>
                  </property>
                </item>
                <item>
                  <property name="text" >
                    <string>14</string>
                  </property>
                </item>
                <item>
                  <property name="text" >
                    <string>16</string>
                  </property>
                </item>
                <item>
                  <property name="text" >
                    <string>20</string>
                  </property>
                </item>
                <item>
                  <property name="text" >
                    <string>24</string>
                  </property>
                </item>
                <item>
                  <property name="text" >
                    <string>32</string>
                  </property>
                </item>
                <item>
                  <property name="text" >
                    <string>48</string>
                  </property>
                </item>
                <item>
                  <property name="text" >
                    <string>64</string>
                  </property>
                </item>
              </widget>
            </item>
            <item row="0" column="0" >
              <widget class="QLabel" name="txtFontsize" >
                <property name="text" >
                  <string>Base font si&amp;ze:</string>
                </property>
                <property name="buddy" stdset="0" >
                  <cstring>basefontsize</cstring>
                </property>
                <property name="wordWrap" >
                  <bool>false</bool>
                </property>
              </widget>
            </item>
            <item rowspan="1" row="1" column="0" colspan="2" >
              <widget class="QCheckBox" name="dontScale" >
                <property name="sizePolicy" >
                  <sizepolicy>
                    <hsizetype>3</hsizetype>
                    <vsizetype>0</vsizetype>
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                  </sizepolicy>
                </property>
                <property name="text" >
                  <string>&amp;Use same size for all elements</string>
                </property>
                <property name="whatsThis" stdset="0" >
                  <string>&lt;b>Use same size for all elements&lt;/b>&lt;p>Select this option to override custom font sizes in favor of the base font size. All fonts will be displayed in the same size.&lt;/p></string>
                </property>
              </widget>
            </item>
          </layout>
        </widget>
      </item>
      <item rowspan="1" row="2" column="0" colspan="2" >
        <widget class="Q3GroupBox" name="GroupBox3_2" >
          <property name="title" >
            <string>Images</string>
          </property>
          <property name="whatsThis" stdset="0" >
            <string>&lt;b>Images&lt;/b>&lt;p></string>
          </property>
          <layout class="QGridLayout" >
            <item row="0" column="0" >
              <widget class="QCheckBox" name="hideImages" >
                <property name="text" >
                  <string>&amp;Suppress images</string>
                </property>
                <property name="whatsThis" stdset="0" >
                  <string>&lt;b>Suppress images&lt;/b>&lt;p>Selecting this will prevent Konqueror from loading images.&lt;/p></string>
                </property>
              </widget>
            </item>
            <item row="1" column="0" >
              <widget class="QCheckBox" name="hideBackground" >
                <property name="text" >
                  <string>Suppress background images</string>
                </property>
                <property name="checked" >
                  <bool>true</bool>
                </property>
                <property name="whatsThis" stdset="0" >
                  <string>&lt;b>Suppress background images&lt;/b>&lt;p>Selecting this option will prevent Konqueror from loading background images.&lt;/p></string>
                </property>
              </widget>
            </item>
          </layout>
        </widget>
      </item>
      <item row="0" column="0" >
        <widget class="Q3GroupBox" name="GroupBox7" >
          <property name="title" >
            <string>Font Family</string>
          </property>
          <property name="whatsThis" stdset="0" >
            <string>&lt;b>Font family&lt;/b>&lt;p>A font family is a group of fonts that resemble one another, with family members that are e.g. bold, italic, or any number of the above.&lt;/p></string>
          </property>
          <layout class="QGridLayout" >
            <item row="0" column="0" >
              <widget class="QLabel" name="txtBaseFontFamily" >
                <property name="text" >
                  <string>Base fa&amp;mily:</string>
                </property>
                <property name="buddy" stdset="0" >
                  <cstring>fontFamily</cstring>
                </property>
                <property name="wordWrap" >
                  <bool>false</bool>
                </property>
              </widget>
            </item>
            <item row="0" column="1" >
              <widget class="QComboBox" name="fontFamily" >
                <property name="sizePolicy" >
                  <sizepolicy>
                    <hsizetype>3</hsizetype>
                    <vsizetype>0</vsizetype>
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                  </sizepolicy>
                </property>
                <property name="whatsThis" stdset="0" >
                  <string>&lt;p>This is the currently selected font family&lt;/p></string>
                </property>
              </widget>
            </item>
            <item row="0" column="2" >
              <spacer name="Spacer7" >
                <property name="sizeHint" >
                  <size>
                    <width>0</width>
                    <height>20</height>
                  </size>
                </property>
                <property name="sizeType" >
                  <enum>Expanding</enum>
                </property>
                <property name="orientation" >
                  <enum>Horizontal</enum>
                </property>
              </spacer>
            </item>
            <item rowspan="1" row="1" column="0" colspan="3" >
              <widget class="QCheckBox" name="sameFamily" >
                <property name="text" >
                  <string>Use same family for all text</string>
                </property>
                <property name="whatsThis" stdset="0" >
                  <string>&lt;b>Use same family for all text&lt;/b>&lt;p>Select this option to override custom fonts everywhere in favor of the base font.&lt;/p></string>
                </property>
              </widget>
            </item>
          </layout>
        </widget>
      </item>
      <item row="4" column="1" >
        <spacer name="Spacer5_2" >
          <property name="sizeHint" >
            <size>
              <width>20</width>
              <height>0</height>
            </size>
          </property>
          <property name="sizeType" >
            <enum>Expanding</enum>
          </property>
          <property name="orientation" >
            <enum>Horizontal</enum>
          </property>
        </spacer>
      </item>
      <item rowspan="1" row="3" column="0" colspan="2" >
        <layout class="QHBoxLayout" >
          <property name="spacing" >
            <number>6</number>
          </property>
          <item>
            <widget class="QPushButton" name="preview" >
              <property name="text" >
                <string>&amp;Preview</string>
              </property>
              <property name="whatsThis" stdset="0" >
                <string>&lt;b>Preview&lt;/b>&lt;p>Click on this button to see what your selections look like in action.&lt;/p></string>
              </property>
            </widget>
          </item>
          <item>
            <spacer name="Spacer1_4" >
              <property name="sizeHint" >
                <size>
                  <width>0</width>
                  <height>20</height>
                </size>
              </property>
              <property name="sizeType" >
                <enum>Expanding</enum>
              </property>
              <property name="orientation" >
                <enum>Horizontal</enum>
              </property>
            </spacer>
          </item>
        </layout>
      </item>
      <item rowspan="1" row="1" column="0" colspan="2" >
        <widget class="Q3ButtonGroup" name="ButtonGroup2" >
          <property name="title" >
            <string>Colors</string>
          </property>
          <property name="exclusive" >
            <bool>false</bool>
          </property>
          <property name="radioButtonExclusive" >
            <bool>true</bool>
          </property>
          <layout class="QVBoxLayout" >
            <item>
              <widget class="QRadioButton" name="blackOnWhite" >
                <property name="text" >
                  <string>&amp;Black on white</string>
                </property>
                <property name="checked" >
                  <bool>true</bool>
                </property>
                <property name="whatsThis" stdset="0" >
                  <string>&lt;b>Black on White&lt;/b>&lt;p>This is what you normally see.&lt;/p></string>
                </property>
              </widget>
            </item>
            <item>
              <widget class="QRadioButton" name="whiteOnBlack" >
                <property name="text" >
                  <string>&amp;White on black</string>
                </property>
                <property name="whatsThis" stdset="0" >
                  <string>&lt;b>White on Black&lt;/b>&lt;p>This is your classic inverse color scheme.&lt;/p></string>
                </property>
              </widget>
            </item>
            <item>
              <widget class="QRadioButton" name="customColor" >
                <property name="text" >
                  <string>Cus&amp;tom</string>
                </property>
                <property name="whatsThis" stdset="0" >
                  <string>&lt;b>Custom&lt;/b>&lt;p>Select this option to define a custom color for the default font.&lt;/p></string>
                </property>
              </widget>
            </item>
            <item>
              <layout class="QGridLayout" >
                <property name="margin" >
                  <number>0</number>
                </property>
                <property name="spacing" >
                  <number>6</number>
                </property>
                <item row="1" column="2" >
                  <widget class="KColorButton" name="foregroundColorButton" >
                    <property name="text" >
                      <string/>
                    </property>
                    <property name="whatsThis" stdset="0" >
                      <string>&lt;b>Foreground color&lt;/b>&lt;p>The foreground color is the color that the text is drawn in.&lt;/p></string>
                    </property>
                  </widget>
                </item>
                <item row="1" column="3" >
                  <spacer name="Spacer9" >
                    <property name="sizeHint" >
                      <size>
                        <width>0</width>
                        <height>20</height>
                      </size>
                    </property>
                    <property name="sizeType" >
                      <enum>Expanding</enum>
                    </property>
                    <property name="orientation" >
                      <enum>Horizontal</enum>
                    </property>
                  </spacer>
                </item>
                <item row="0" column="0" >
                  <spacer name="Spacer1" >
                    <property name="sizeHint" >
                      <size>
                        <width>21</width>
                        <height>20</height>
                      </size>
                    </property>
                    <property name="sizeType" >
                      <enum>Fixed</enum>
                    </property>
                    <property name="orientation" >
                      <enum>Horizontal</enum>
                    </property>
                  </spacer>
                </item>
                <item row="1" column="0" >
                  <spacer name="Spacer1_2" >
                    <property name="sizeHint" >
                      <size>
                        <width>20</width>
                        <height>20</height>
                      </size>
                    </property>
                    <property name="sizeType" >
                      <enum>Fixed</enum>
                    </property>
                    <property name="orientation" >
                      <enum>Horizontal</enum>
                    </property>
                  </spacer>
                </item>
                <item row="0" column="3" >
                  <spacer name="Spacer8" >
                    <property name="sizeHint" >
                      <size>
                        <width>0</width>
                        <height>20</height>
                      </size>
                    </property>
                    <property name="sizeType" >
                      <enum>Expanding</enum>
                    </property>
                    <property name="orientation" >
                      <enum>Horizontal</enum>
                    </property>
                  </spacer>
                </item>
                <item row="1" column="1" >
                  <widget class="QLabel" name="txtForeground" >
                    <property name="text" >
                      <string>&amp;Foreground:</string>
                    </property>
                    <property name="buddy" stdset="0" >
                      <cstring>foregroundColorButton</cstring>
                    </property>
                    <property name="whatsThis" stdset="0" >
                      <string>&lt;b>Foreground color&lt;/b>&lt;p>The foreground color is the color that the text is drawn in.&lt;/p></string>
                    </property>
                    <property name="wordWrap" >
                      <bool>false</bool>
                    </property>
                  </widget>
                </item>
                <item row="0" column="2" >
                  <widget class="KColorButton" name="backgroundColorButton" >
                    <property name="text" >
                      <string/>
                    </property>
                    <property name="whatsThis" stdset="0" >
                      <string>&lt;b>Background&lt;/b>&lt;p>Behind this door lays the ability to choose a custom default background.&lt;/p></string>
                    </property>
                  </widget>
                </item>
                <item row="0" column="1" >
                  <widget class="QLabel" name="txtBackground" >
                    <property name="text" >
                      <string>Bac&amp;kground:</string>
                    </property>
                    <property name="buddy" stdset="0" >
                      <cstring>backgroundColorButton</cstring>
                    </property>
                    <property name="whatsThis" stdset="0" >
                      <string>&lt;b>Background&lt;/b>&lt;p>This background color is the one displayed behind the text by default. A background image will override this.&lt;/p></string>
                    </property>
                    <property name="wordWrap" >
                      <bool>false</bool>
                    </property>
                  </widget>
                </item>
              </layout>
            </item>
            <item>
              <widget class="QCheckBox" name="sameColor" >
                <property name="text" >
                  <string>Use same color for all text</string>
                </property>
                <property name="whatsThis" stdset="0" >
                  <string>&lt;b>Use same color for all text&lt;/b>&lt;p>Select this option to apply your chosen color to the default font as well as any custom fonts as specified in a stylesheet.&lt;/p></string>
                </property>
              </widget>
            </item>
          </layout>
        </widget>
      </item>
    </layout>
  </widget>
  <layoutdefault spacing="6" margin="11" />
  <layoutfunction spacing="KDialog::spacingHint" margin="KDialog::marginHint" />
  <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
  <tabstops>
    <tabstop>fontFamily</tabstop>
    <tabstop>sameFamily</tabstop>
    <tabstop>basefontsize</tabstop>
    <tabstop>dontScale</tabstop>
    <tabstop>blackOnWhite</tabstop>
    <tabstop>backgroundColorButton</tabstop>
    <tabstop>foregroundColorButton</tabstop>
    <tabstop>sameColor</tabstop>
    <tabstop>hideImages</tabstop>
    <tabstop>hideBackground</tabstop>
    <tabstop>preview</tabstop>
  </tabstops>
  <includes>
    <include location="global" >kcolorbutton.h</include>
    <include location="local" >kcolorbutton.h</include>
  </includes>
</ui>
